<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>20</width>
      <height>631</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>45</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>图书查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bookin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>425</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>图书入库</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnbook">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>还书管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lendbook">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>235</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>借书管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manage">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>借书证管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>管理员登陆</string>
    </property>
   </widget>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>635</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>640</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>1051</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Cardmanage">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>681</width>
        <height>541</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>添加借书证</string>
       </attribute>
       <widget class="QLineEdit" name="addcardText">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>511</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>311</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>请输入要添加的借书证号</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addcard">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>380</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>添加</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>单位</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>310</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>类别</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cardName">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>180</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="cardDepart">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>240</y>
          <width>241</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="cardType">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>310</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string>学生卡</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>学生卡</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>教师卡</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>删除借书证</string>
       </attribute>
       <widget class="QLineEdit" name="deletcardText">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>200</y>
          <width>511</width>
          <height>61</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>110</y>
          <width>311</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>请输入要删除的借书证号</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deletecard">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>350</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>所有借书证</string>
       </attribute>
       <widget class="QTableView" name="Allcard">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>681</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="refresh">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>470</y>
          <width>71</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Bookinpage">
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>851</width>
        <height>581</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>图书入库</string>
       </attribute>
       <widget class="QLineEdit" name="booknotext">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>30</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="booknolabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>书号</string>
        </property>
       </widget>
       <widget class="QLabel" name="booktypelabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>图书类别</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="booktypetext">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>80</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="booknametext">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>120</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="booknamelabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>书名</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="publishertext">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>160</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="publisherlabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>160</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>出版社</string>
        </property>
       </widget>
       <widget class="QLabel" name="pubyearlabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>200</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>出版年份</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pubyeartext">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>200</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="bookpricelabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>280</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>单价</string>
        </property>
       </widget>
       <widget class="QLabel" name="authorlabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>240</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>作者</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="authortext">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>240</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="pricetext">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>280</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addBook">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>410</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>添加图书</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="booknumtext">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>320</y>
          <width>341</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="booknumlabel">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>320</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>数量</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>已有图书</string>
       </attribute>
       <widget class="QTableView" name="BookinfoTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>851</width>
          <height>541</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="searchPage">
     <widget class="QLabel" name="authorlabel_2">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>作者</string>
      </property>
     </widget>
     <widget class="QLabel" name="publisherlabel_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>出版社</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pubyeartextlow">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pubyearlabel_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>出版年份</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchnametext">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="bookpricelabel_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>价格范围</string>
      </property>
     </widget>
     <widget class="QLabel" name="booknamelabel_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>书名</string>
      </property>
     </widget>
     <widget class="QLabel" name="booktypelabel_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>图书类别</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchpublishertext">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="pricetextlow">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="booktypetext_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchauthortext">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="pricetexthigh">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>60</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="pubyeartexthigh">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>60</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>04b_21</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>70</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>04b_21</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTableView" name="searchtable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>971</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchbutton">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>10</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearbutton">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>60</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lendbookpage">
     <widget class="QLineEdit" name="lendcardno">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>72</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>借书证号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>72</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>书号</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lendbookno">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="lendbookbutton">
      <property name="geometry">
       <rect>
        <x>782</x>
        <y>70</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>借书</string>
      </property>
     </widget>
     <widget class="QPushButton" name="lendcheckbutton">
      <property name="geometry">
       <rect>
        <x>652</x>
        <y>70</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查询已借图书</string>
      </property>
     </widget>
     <widget class="QTableView" name="lendbooktable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>921</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="returnbookpage">
     <widget class="QTableView" name="returnbooktable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>921</width>
        <height>511</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="returnbookno">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>72</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>借书证号</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>72</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>书号</string>
      </property>
     </widget>
     <widget class="QPushButton" name="returnbookbutton">
      <property name="geometry">
       <rect>
        <x>782</x>
        <y>60</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>还书</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="returncardno">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="returncheckbutton">
      <property name="geometry">
       <rect>
        <x>652</x>
        <y>60</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>查询已借图书</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
